libs/platform/src/lib/components/object-status/object-status.component.ts

```diff
@@ -0,0 +1,45 @@
+import { Component, Input } from '@angular/core';
+import { ObjectStatus } from '@fundamental-ngx/core';
+
+@Component({
+    selector: 'fdp-object-status',
+    templateUrl: './object-status.component.html'
+})
+export class ObjectStatusComponent {
+    /**
+     * The status represented by the Object Status.
+     * Can be one of the following: 'negative' | 'critical' | 'positive' | 'informative'
+     * For default Object Status omit this property
+     */
+    @Input()
+    status: ObjectStatus;
+
+    /**
+     * Glyph (icon) of the Object Status.
+     */
+    @Input()
+    glyph: string;
+
+    /**
+     * A number representing the indication color.
+     * Option includes numbers from 1 to 8
+     */
+    @Input()
+    indicationColor: number = null;
```

Comment:
I know core has the `indicationColor` as a `number` type, but I think we should introduce a `IndicationColor` type here since only integers 1 through 8 are valid choices, e.g.
```
export type IndicationColor = 1 | 2 | 3 | 4 | 5 | 6 | 7 | 8;
```